README.md

```diff
@@ -29,32 +29,60 @@ ember install ember-frost-modal-input
 | `title` | `string` | | Optional custom title |
 | `subtitle` | `string` | |  Optional custom subtitle |
 | `titleComponent` | `string` | | Optional title component to replace standard title/subtitle styles |
+| `modalName` | `string` | | Optional name for your modal |
 
 ## Examples
 
-### Parent Controller
-Inject the modal-forms service into the parent controller
-
+### Modal component
+Inject the [remodal](http://sethbrasile.github.io/ember-remodal/) service in your component
 ```js
-modalForms: Ember.inject.service('modal-forms'),
-isModalActive: Ember.computed.readOnly('modalForms.isModalActive'),
+remodal: inject.service(),
 ```
 
-Change the active state of the modal in your component
-```js
-willInsertElement () {
-  this.get('modalForms').setModalActive(true)
-},
-
-willDestroyElement () {
-  this.get('modalForms').setModalActive(false)
-}
-```
-
-### Parent Component Template
-Custom classes can be applied to the modal template, based on state of the modal
+#### Component template using ember-block-slots
+Below is an example of template.hbs
 ```handlebars
-{{liquid-modal class=(if isModalActive 'form-container' '')}}
+{{#frost-modal-input
+  modalName=modalName as |slot|}}
+  {{#block-slot slot 'openButton'}}
+    {{frost-button
+      text='Open long form with scroll'
+      priority='secondary'
+      size='medium'}}
+  {{/block-slot}}
+
+  {{#block-slot slot 'header'}}
+    {{#frost-info-bar as |header|}}
+      {{#block-slot header 'title'}}
+        Edit user
+      {{/block-slot}}
+      {{#block-slot header 'summary'}}
+        John Smith
+      {{/block-slot}}
+    {{/frost-info-bar}}
+  {{/block-slot}}
+
+  {{#block-slot slot 'form'}}
```

Comment:
If we aren't tying directly to bunsen (e.g. yielding a form or taking one in the root params) then `content` is probably a better fit